linkerd/service-profiles/src/lib.rs

```diff
@@ -26,7 +27,15 @@ pub mod split;
 
 pub use self::client::Client;
 
-pub type Receiver = tokio::sync::watch::Receiver<Profile>;
+#[derive(Clone, Debug)]
+pub struct Receiver {
+    inner: tokio::sync::watch::Receiver<Profile>,
+}
+
+#[derive(Debug)]
+struct ReceiverStream {
+    inner: tokio_stream::wrappers::WatchStream<Profile>,
+}
```

Comment:
does this wrapper do anything? AFAICT it could just be a type alias for `WatchStream<Profile>`...is the intent to hang more functionality off the stream type in the future?

not a blocker.

Reply:
Yeah. For the same reason we have the `Receiver` wrapper -- we'll need to be able to hold a handle as long as the stream is active.